### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by tapadipti**
> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

File: content/docs/command-reference/import.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ working directory with its original file name e.g. `data.txt` (or to a location
 provided with `--out`). An _import `.dvc` file_ is created in the same location
 e.g. `data.txt.dvc` – similar to using `dvc add` after downloading the data.
 
-(ℹ️) DVC won't push data imported from other DVC repos to
+ℹ️ DVC won't push data imported from other DVC repos to
 [remote storage](/doc/command-reference/remote). `dvc pull` will download from
 the original source.
```
Comment (daavoo) [inline content/docs/command-reference/import.md:43]: We can now use admonition blocks (https://github.com/iterative/dvc.org/issues/3382):

```
<admon type="info">

DVC won't push data imported from other DVC repos to
[remote storage](/doc/command-reference/remote). `dvc pull` will download from
the original source.

</admon>
```

Comment (tapadipti) [reply content/docs/command-reference/import.md:43]: How do I make the links / code block work? Currently, it looks like this:
image

Comment (daavoo) [reply content/docs/command-reference/import.md]: It seems to be that the empty lines must be present:
```
# Doesn't render links
<admon type="info">
DVC won't push data imported from other DVC repos to
[remote storage](/doc/command-reference/remote). `dvc pull` will download from
the original source.
</admon>

# Does render links

<admon type="info">

DVC won't push data imported from other DVC repos to
[remote storage](/doc/command-reference/remote). `dvc pull` will download from
the original source.

</admon>

Comment (daavoo) [reply content/docs/command-reference/import.md]: I pushed changes with empty lines and links do render properly

Comment (tapadipti) [reply content/docs/command-reference/import.md]: Oh.. Thanks @daavoo

**Review by daavoo — APPROVED** [review]

File: content/docs/command-reference/import.md (hunk 1 above)
Comment (tapadipti) [reply content/docs/command-reference/import.md]: A question: I don't think that we're using admonition blocks in many parts in the doc. Won't this look inconsistent?

Comment (daavoo) [reply content/docs/command-reference/import.md]: The idea is to migrate all docs. 

IMO it's a good strategy to just add an admonition block whenever you happen to stump upon one of the old usages, not sure what the rest of @iterative/docs team think

**Comment by iesahin** [comment]
We are considering to move everything to admonitions block wherever we
update. Thank you @tapadipti @daavoo

File: content/docs/command-reference/import.md (hunk 1 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/import.md]: > it's a good strategy to just add an admonition block whenever you happen to stump upon one

Yup that's the idea. Not a huge priority to migrate all at once rn

**Pull request merged**